parser/proto.y

```diff
@@ -488,16 +488,26 @@ listLiteral : '[' listElements ']' {
 		$$ = ast.NewArrayLiteralNode($1, nil, nil, $2)
 	}
 	| '[' error ']' {
-		$$ = nil
+		$$ = ast.NewArrayLiteralNode($1, nil, nil, $3)
```

Comment:
I added this as "just in case". But the only place this production (`listLiteral`) is used is in the `value` production. And the only place that is used is in `messageLiteralField`, which is where I added the two new nil checks above. But it still seems reasonable to keep this -- this way we do get a value in the AST, but it's an empty list (instead of eliding this field from the message literal AST node entirely).